src/mainwindow.cpp

```diff
@@ -1273,15 +1285,17 @@ void MainWindow::restoreStates()
         m_splitter->restoreState(
                 m_settingsDatabase->value(QStringLiteral("splitterSizes")).toByteArray());
 
+    if (m_settingsDatabase->value(QStringLiteral("isTreeCollapsed"), "NULL") != "NULL") {
```

Comment:
I think we can just remove these "NULL" checks, since the behavior will be exactly the same without them (if there is no value set, the `toBool` will return `false`.